src/ui/screens/home/controller/ControllerView.qml

```diff
@@ -477,19 +477,18 @@ Item {
 
         onClicked: {
             if (!box.connectionInfoScreenVisible) {
-                Glean.interaction.speedTestTriggered.record({
+                Glean.interaction.startSpeedTestSelected.record({
                     screen: "main",
-                    action: "select",
-                    element_id: "speed_test"
                 });
             } else {
-                Glean.interaction.speedTestClosed.record({
-                    screen: connectionInfoScreen.state == "open-ready" ? "speed_test_result"
+                Glean.interaction.closeSelected.record({
+                    screen: connectionInfoScreen.state == "open-ready" ? "speed_test_completed"
                                 : connectionInfoScreen.state == "open-error" ? "speed_test_error"
                                     : connectionInfoScreen.state == "open-loading" ? "speed_test_loading"
```

Comment:
Shouldn't speed_test_error & speed_test_loading be added to outcore_metrics.yaml?

Reply:
Nope, these are interaction metrics. Note that, for the metrics definition I am just following what was already designed by Santiago and Yeonjoo, see: https://miro.com/app/board/uXjVM_QZzjA=/.

Reply:
I see speed_test_completed in outcome_metrics.yaml. Do speed_test_error and speed_test_loading be needed to added to any yaml file?

Reply:
Ah, ok I think I see where the confusion is coming from. In the lines above we are setting the extra key "screen" for the "close_selected" metric. The Miro board specifies, that "speed_test_completed", "speed_test_error" and "speed_test_loading" are the possible values for that extra key in this metric. 

The "speed_test_completed" metric is something different, and is the only outcome event defined in that miro board. The key here is the difference between metric names and extra key values.

Notem that on the `metrics.yaml` files the metric names and extra keys are defined, but the value of extra key property is not defined. Let me know if this makes sense.

Reply:
Got it, thanks.